db/primary_migrate/20230803143215_add_mr_chipertext_columns.rb

```diff
@@ -0,0 +1,9 @@
+class AddMrChipertextColumns < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :encrypted_password_digest_mr, :string
```

Comment:
what do you think of expanding it to `multi_region` instead of `mr` for clarity?

Reply:
I am happy to

Reply:
see https://github.com/18F/identity-idp/pull/8931#discussion_r1283401052